### PR TITLE
ci: run the bundle promotion test

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

**Comment by lengau** [comment]
Issue created about juju: https://github.com/canonical/charmcraft/issues/1096

File: .github/workflows/spread.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ jobs:
         spread:
           - "google:"
           - "google:ubuntu-22.04-64:tests/spread/charms/k8s-operator:prometheus"
+          - "google:ubuntu-22.04-64:tests/spread/charms/bundle:notebook_operators"
```
[the remark this answers is not in the document]
Comment (lengau) [reply .github/workflows/spread.yaml:36]: That makes sense - unlike the other items in this directory, this isn't a particularly resource-intensive test (at least, not at this point - that could change if we add other bundles). 

I've made this test automatic and created #1097 for reorganising the large spread tests

[the remark this answers is not in the document]
Comment (mr-cal) [reply .github/workflows/spread.yaml:36]: If it's automatic, then can this line be removed?

[the remark this answers is not in the document]
Comment (sergiusens) [reply .github/workflows/spread.yaml:36]: this line should be removed
I will

**Review by mr-cal — APPROVED** [review]

**Pull request merged**